Source/Scene/TileBoundingRegion.js

```diff
@@ -41,6 +45,16 @@ define([
         this.minimumHeight = defaultValue(options.minimumHeight, 0.0);
         this.maximumHeight = defaultValue(options.maximumHeight, 0.0);
 
+
+        /**
+         * An oriented bounding box that encloses this tile's region.
+         * This is used to calculate tile visibility.
+         *
+         * @type {OrientedBoundingBox}
+         * @default OrientedBoundingBox()
+         */
+        this.orientedBoundingBox = new OrientedBoundingBox();
```

Comment:
Can this be private?  Just change it to:
```
this._orientedBoundingBox = new OrientedBoundingBox();
```
Also, I would just inline the `computeOrientedBoundingBox` function since it is awkward that we allocate the object here, then allocate it again.